.github/ISSUE_TEMPLATE/dataset_task.md

```diff
@@ -4,6 +4,7 @@ about: Create a ticket for a dataset-related wrangler task.
 title: dataset problem - HCADataSetXYZ
 labels: dataset, operations, task
 assignees: ''
+projects: "ebi-ait/12"
```

Comment:
shouldn't this be `ebi-ait/11` ? It's supposed to go on the operations board

Reply:
thanks for fishing this out!